django_extractors/settings.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from django_path import DJANGO_PATH
+
+settings_path = os.path.join(DJANGO_PATH, 'conf', 'global_settings.py')
+variables = ''
+
+with open(settings_path, 'r') as settings_file:
+    for line in settings_file:
+        if not line.startswith('#'):
+            if '=' in line:
+                split_line = line.split(' = ')
```

Comment:
What if someone has a setting like:

    MY_SETTING=True

Could we more conservatively split on just the equals-sign? Then again, that might be too general because I think I've seen an equals sign end up in the secret key Django has generated. We'd need some fancy regex stuff then to handle that.

Though, the linting rules should point out the lack of whitespace around the assignment operator anyway. Don't mind me, just thinking out loud :)